### PR TITLE
To Do List structure

### PR DESCRIPTION
**Pull request by yuvenalmash**
# The following changes were made:
- created the UI structure of the To DO application
- updated the readme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^7.1.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (MaganaAsiati) [inline package.json]: - Kindly make sure you did install all dependencies and configured correctly Webpack. when running npm run start you are able to see the interface. Make sure those two points I list below are correctly implemented. follow this [link](https://webpack.js.org/guides/getting-started/) on  how to use Webpack  

- Webpack is used to set up the project. Basic commands for building Webpack in development mode and production mode are configured (check the package.json script or readme instruction for this). The Webpack dev server is set up and running with no errors. Make sure the error is not from your machine. :+1: 


- It has an HTML file configured in the Webpack config settings as a template using the HTML Webpack Plugin :+1: .

File: toDoList/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <title>To Do List</title>
+<script defer src="/runtime.bundle.js"></script><script defer src="/main.bundle.js"></script></head>
+<body>
+    <div id="container">
+        <h2>Today's To Do</h2>
+        <hr>
+        <input id="inputToDo" placeholder="Add to your list..." type="text">
+        <hr>
+        <div id="toDos">
+
```
Comment (MaganaAsiati) [inline toDoList/dist/index.html]: - Kindly make sure On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task. As you see in the picture below there is no task appears:+1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @jossi30, I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- Kindly make sure that you use the Webpack and when loading the page in the user interface there is a list of the task as is required in the project. as see I think you have another To do list  project inside your project reason why the configuration for webpack is not working cause you have two projects combine kindly remove the todo list folder inside your project and restart all configuration it will help you to fix the bag you have :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        </div>
+        <hr>
+        <div id="toDos">
+        </div>
+        <footer>
+            <a href="#">Clear all completed</a>
+        </footer>
+
+    </div>
+</body>
```
Comment (Kamba56) [inline src/index.html:25]: - [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also best practice.

**Review by Kamba56 — APPROVED** [review]
Hi @yuvenalmash, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**